extensions/azuremonitor/src/ui-references.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Please use the standard copyright header for this. I'm kind of surprised it let you commit with this actually - did you not get an error from the precommit hooks?